package.json

```diff
@@ -21,33 +21,40 @@
   "scripts": {
     "coverage": "istanbul cover _mocha -x src/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
     "start": "node example/server.js",
-    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "test": "eslint . && mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
```

Comment:
I think that we should keep separating the linting step from the test one; I'd rather create another npm script and maybe integrate them both w/ a pre-commit hook